reflekt-plugin/src/main/kotlin/io/reflekt/plugin/analysis/ir/ArgumentCollectors.kt

```diff
@@ -0,0 +1,69 @@
+package io.reflekt.plugin.analysis.ir
+
+import io.reflekt.plugin.analysis.common.ReflektFunction
+import io.reflekt.plugin.analysis.models.*
+import org.jetbrains.kotlin.ir.IrElement
+import org.jetbrains.kotlin.ir.expressions.*
+import org.jetbrains.kotlin.ir.visitors.IrElementVisitor
+
+class ReflektInvokeArgumentsCollector : IrElementVisitor<Unit, Nothing?> {
```

Comment:
Please, add some description for base classes such as ReflektInvokeArgumentsCollector, ReflektIrInvokesAnalyzer, ReflektIrUsesAnalyzer